src/models/converters/transactions/tests/summary.rs

```diff
@@ -11,12 +11,12 @@ use crate::models::commons::ParamValue::SingleValue;
 
 #[test]
 fn data_size_calculation() {
-    assert_eq!(data_size(&None), "0");
```

Comment:
How often do we test this method?

Reply:
I think too often. But it is usually tested indirectly. I wanted to have something where we asserted against `None`, `0`, `0x` and happy path. It could be a little too paranoid.

Reply:
We test this directly at least 3 times (visible in this pr)

Reply:
You are right, I removed the redundant ones.